compiler/ast.nim

```diff
@@ -494,6 +494,8 @@ type
     tfGenericTypeParam
     tfImplicitTypeParam
     tfInferrableStatic
+    tfExplicit        # for typedescs, marks types explicitly prefixed with the
```

Comment:
So which part of the spec this deals with? I don't like it. :-) Why is ``type x`` suddenly different from e.g.  ``typeDesc[int]``?